src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,841 @@
+    Subnets:
+      100.2.0.0/16
+      10.2.0.0/16
+  Status Failure:  
+  Version:         v0.8.0-pre0-1-g5d7f163
+Events:            <none>
+```
+<!-- markdownlint-enable no-trailing-spaces -->
+To verify that service discovery is installed properly, check that the `submariner-lighthouse-coredns` Service is ready:
+
+```bash
+$ kubectl -n submariner-operator get service submariner-lighthouse-coredns
+NAME                            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+submariner-lighthouse-coredns   ClusterIP   100.2.177.123   <none>        53/UDP    126m
+```
+
+Next, verify that CoreDNS was properly configured to forward requests sent to `clusterset.local` to Lighthouse CoreDNS Server in the
+cluster:
+
+```bash
+$ kubectl -n kube-system describe configmap coredns
+Name:         coredns
+Namespace:    kube-system
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+Corefile:
+----
+#lighthouse-start AUTO-GENERATED SECTION. DO NOT EDIT
+clusterset.local:53 {
+    forward . 100.2.177.123
+}
+#lighthouse-end
+.:53 {
+    errors
+    health {
+       lameduck 5s
+    }
+    ready
+    kubernetes cluster2.local in-addr.arpa ip6.arpa {
+       pods insecure
+       fallthrough in-addr.arpa ip6.arpa
+       ttl 30
+    }
+    prometheus :9153
+    forward . /etc/resolv.conf
+    cache 30
+    loop
+    reload
+    loadbalance
+}
+```
+
+Note that **100.2.177.123** is the ClusterIP address of the `submariner-lighthouse-coredns` Service we verified earlier.
+
+### 2. Export Services Across Clusters
+
+At this point, we have enabled secure IP communication between the connected clusters. However, further configuration is required in order
+to signify that a Service should be visible and discoverable to other clusters in the clusterset. This can be done by creating a
+`ServiceExport` object in each cluster within the namespace that the underlying Service resides in. When a `ServiceExport` is created, this
+will cause the multi-cluster Service to become accessible as `<service>.<ns>.svc.clusterset.local`. Similarly, deleting the `ServiceExport`
+will stop exporting the Service.
+
+This guide uses a simple nginx server for demonstration purposes.
+
+#### Test ClusterIP Service
+
+A Kubernetes Service is an abstraction which defines a logical set of Pods running in your cluster, that all provide the same functionality.
+When created, each Service is assigned a unique IP address. This address is tied to the lifespan of the Service.
+Pods can be configured to talk to the Service, and know that communication to the Service will be automatically load-balanced out to some
+Pod that is a member of the Service. ClusterIP is the default ServiceType in Kubernetes, and is only accessible from within the cluster.
+Submariner makes it possible for a ClusterIP Service to be accessible across cluster boundaries.
+
+##### 1. Create an `nginx` Pod on **cluster3**
+
+Create a Pod with the following Deployment specification:
```

Comment:
Shall we may be use the simple cmdlines we have on the quickstart?, this would limit the focus we put on deploying the pod, which is less important to this guide.

Let's also use the unprivileged nginx.